config/config.json

```diff
@@ -17,11 +17,12 @@
     "operatorsAliases": false
   },
   "production": {
```

Comment:
can we add a `TODO` here just as a reminder to change this to actual prod values once we have that set up

Reply:
Hmm apparently you can't add comments to json files... can we make a note on our notion task board instead?

Reply:
whoops my bad, i'll put it on a ticket instead